### PR TITLE
Codex lore-text changes to fit Foundation lore

### PR DESCRIPTION
**Pull request by aidan-bowman**
## About the Pull Request

The codex hasn't been updated all that much, which leaves a lot of references to bay lore still up. I tried to refit as many as I could, although it's hard to convert many of the descriptions (especially considering how much SCP leaves undefined).

Oh yeah, and I think I changed the description of the chameleon energy gun. I don't know if that's even used anywhere.

## Why It's Good For The Game

TBH I prefer the Codex over having a dedicated wiki. It's a lot faster to read the codex entry for an item than search for it on an external browser, and since the Codex reads out of the actual files, it's self-updating. Plus, it's much easier for a coder to simply edit a couple descriptions in their PR then wait for it to get merged and use Wikidot to write an entire entry.

I'm hoping this PR produces more coding effort to fixing up the Codex in general.

## Potential improvements

A large amount of the descriptions I wrote are made up and low-quality, so there's probably loads of minor changes that would improve the PR.

## Changelog

:cl:
fix: Massive changes to the lore-text in the Codex.
fix: Changed chameleon energy gun description to better fit SCP lore.
/:cl:

### CODE REVIEW

File: code/modules/codex/categories/category_species.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /datum/codex_category/species/
 	name = "Species"
-	desc = "Sapient species encountered in known space."
+	desc = "Anomalous species that show signs of sapience."
```
Comment (MACIEKBAKI) [inline code/modules/codex/categories/category_species.dm]: I guess humans are anomalous too

**Comment by MACIEKBAKI** [comment]
This is a really good PR, fixing up the Codex - which, in and of itself, is really a good idea, I mean, an in-game encyclopedia? Don't even have to open a separate browser window - to be more lore-accurate, hopefully some lore nerds can look over this

File: code/modules/codex/entries/codex.dm [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+	lore_text = "The Codex is a widely-used encyclopedia maintained and distributed by the Foundation, for the Foundation. The intent of this document is to arm all staff with basic information of mundane and anomalous objects they might encounter. \
 	<br><br> \
-	Access to the Codex is afforded instantly by a variety of unobtrusive devices, including PDA attachments, retinal implants, neuro-memetic indoctrination and hover-drone. All our access devices are affordable, stylish and guaranteed not to expose you to encephalo-malware. \
+	Access to the Codex is transferred to all hired personnel via a memetically-spread and nomenclaturely-updated concept. \
 	<br><br> \
-	The Codex editorial offices are located in Venus orbit and will offer cash payments for stories, anecdotes and evidence of the strange and undocumented miscellany cluttering up our observable universe. Remember our motto: <i>\"Knowledge is power, sell your excess.\"</i>"
+	It has been brought to the attention of the Overseer Council that several individuals and groups have altered the documentation system for malicious use. Please report any potential misuse to Security.</i>"
```
Comment (MACIEKBAKI) [inline code/modules/codex/entries/codex.dm]:
```suggestion
	It has been brought to the attention of the Overseer Council that several individuals and groups have altered the documentation system for malicious use. Please report any potential misuse to Security."
```
remove excess closing tag, causes tests to fail

File: code/modules/codex/categories/category_species.dm (hunk 1 above)
Comment (aidan-bowman) [reply code/modules/codex/categories/category_species.dm]: Noted and fixed (sort of)

**Review by MACIEKBAKI — APPROVED** [review]

**Pull request merged**